doc/specifications/proposed/content_download/content_download.md

```diff
@@ -0,0 +1,44 @@
+# Content download
+
+> Author: bd@ez.no
+
+## Summary
+This feature covers downloading of Content binary files over HTTP.
+
+It covers:
+- generation of proper links from Twig as well as PHP,
+- permissions checking (the user must be allowed to read the Content)
+- streaming of the file over HTTP
+
+## Components
+
+### Twig helper
+
+> Name: ez_download_path
+> Usage: `ez_download_path( content, 'file' )`
```

Comment:
As discussed, if you use a [RouteReference](https://doc.ez.no/display/EZP/RouteReference), there will be no need of a twig helper :wink: . You'll just have to implement a listener.